src/explore-education-statistics-admin/src/pages/release/datablocks/components/chart/__tests__/ChartMapCustomGroupsConfiguration.test.tsx

```diff
@@ -221,25 +213,137 @@ describe('ChartMapCustomGroupsConfiguration', () => {
       'cell',
     )[1];
 
+    await waitFor(() => {
+      expect(
+        within(minCell).getByText('Groups cannot overlap'),
+      ).toBeInTheDocument();
+      expect(
+        within(maxCell).queryByText('Groups cannot overlap'),
+      ).not.toBeInTheDocument();
+    });
+  });
+
+  test('shows a validation error when the maximum value is in an existing group', async () => {
+    render(
+      <ChartMapCustomGroupsConfiguration
+        groups={testGroups}
+        id="testId"
+        onAddGroup={noop}
+        onRemoveGroup={noop}
+      />,
+    );
+
+    userEvent.type(screen.getByLabelText('Min'), '-10');
+    userEvent.type(screen.getByLabelText('Max'), '75');
     userEvent.tab();
 
+    const minCell = within(screen.getAllByRole('row')[3]).getAllByRole(
+      'cell',
+    )[0];
+    const maxCell = within(screen.getAllByRole('row')[3]).getAllByRole(
+      'cell',
+    )[1];
+
+    await waitFor(() => {
+      expect(
+        within(minCell).queryByText('Groups cannot overlap'),
+      ).not.toBeInTheDocument();
+
+      expect(
+        within(maxCell).getByText('Groups cannot overlap'),
+      ).toBeInTheDocument();
+    });
+  });
+
+  test('shows validation errors when both values are in an existing group', async () => {
+    render(
+      <ChartMapCustomGroupsConfiguration
+        groups={testGroups}
+        id="testId"
+        onAddGroup={noop}
+        onRemoveGroup={noop}
+      />,
+    );
+
+    userEvent.type(screen.getByLabelText('Min'), '10');
+    userEvent.type(screen.getByLabelText('Max'), '75');
+    userEvent.tab();
+
+    const minCell = within(screen.getAllByRole('row')[3]).getAllByRole(
+      'cell',
+    )[0];
+    const maxCell = within(screen.getAllByRole('row')[3]).getAllByRole(
+      'cell',
+    )[1];
+
     await waitFor(() => {
       expect(
         within(minCell).getByText('Groups cannot overlap'),
       ).toBeInTheDocument();
+      expect(
+        within(maxCell).getByText('Groups cannot overlap'),
+      ).toBeInTheDocument();
```

Comment:
Same as earlier - would make these errors more specific to the field e.g. 'Min cannot overlap another group' and 'Max cannot overlap another group'

Same applies for the other test cases